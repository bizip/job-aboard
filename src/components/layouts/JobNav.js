import React from 'react';
import { Link } from 'react-router-dom';
import classes from './NavBar.module.css';
import microphone from '../../img/microphone.svg';
import setting from '../../img/setting.svg';
import back from '../../img/back.svg';

const propTypes = {};

const defaultProps = {};

const JobNav = () => (
  <nav className={classes.nav}>
    <Link to="/">
      {' '}
      <div className={classes.menuContainer}>
        <img src={back} alt="logo" />
      </div>
    </Link>

    <div className={classes.textContainer}>
      <p>Job aboard</p>
    </div>
    <div className={classes.iconsContainer}>
      <img src={microphone} alt="microphone" />
      <img src={setting} alt="settion" />
    </div>
  </nav>
);

JobNav.propTypes = propTypes;
JobNav.defaultProps = defaultProps;
// #endregion

export default JobNav;
