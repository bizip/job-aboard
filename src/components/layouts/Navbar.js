import React from 'react';
import classes from './NavBar.module.css';
import microphone from '../../img/microphone.svg';
import setting from '../../img/setting.svg';
import plane from '../../img/plane.svg';

const propTypes = {};

const defaultProps = {};

const Navbar = () => (
  <nav className={classes.nav}>
    <div className={classes.menuContainer}>
      <img src={plane} alt="logo" />
    </div>
    <div className={classes.textContainer}>
      <p>Job aboard</p>
    </div>
    <div className={classes.iconsContainer}>
      <img src={microphone} alt="microphone" />
      <img src={setting} alt="settion" />
    </div>
  </nav>
);

Navbar.propTypes = propTypes;
Navbar.defaultProps = defaultProps;
// #endregion

export default Navbar;
