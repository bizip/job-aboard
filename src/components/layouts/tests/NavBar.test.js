import React from 'react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import Navbar from '../Navbar';
import JobNav from '../JobNav';

describe('Test for the navigation bar section', () => {
  it('It should render the navigation bar', () => {
    const nav = renderer
      .create(<Router><Navbar /></Router>)
      .toJSON();
    expect(nav).toMatchSnapshot();
  });

  it('It should render the descripttions navigation bar', () => {
    const nav = renderer
      .create(<Router><JobNav /></Router>)
      .toJSON();
    expect(nav).toMatchSnapshot();
  });
});
