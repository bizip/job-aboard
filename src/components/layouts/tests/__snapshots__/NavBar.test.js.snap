// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for the navigation bar section It should render the descripttions navigation bar 1`] = `
<nav
  className="nav"
>
  <a
    href="#/"
    onClick={[Function]}
  >
     
    <div
      className="menuContainer"
    >
      <img
        alt="logo"
        src="back.svg"
      />
    </div>
  </a>
  <div
    className="textContainer"
  >
    <p>
      Job aboard
    </p>
  </div>
  <div
    className="iconsContainer"
  >
    <img
      alt="microphone"
      src="microphone.svg"
    />
    <img
      alt="settion"
      src="setting.svg"
    />
  </div>
</nav>
`;

exports[`Test for the navigation bar section It should render the navigation bar 1`] = `
<nav
  className="nav"
>
  <a
    href="#/"
    onClick={[Function]}
  >
     
    <div
      className="menuContainer"
    >
      <img
        alt="logo"
        src="plane.svg"
      />
    </div>
  </a>
  <div
    className="textContainer"
  >
    <p>
      Job aboard
    </p>
  </div>
  <div
    className="iconsContainer"
  >
    <img
      alt="microphone"
      src="microphone.svg"
    />
    <img
      alt="settion"
      src="setting.svg"
    />
  </div>
</nav>
`;
